cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -1010,11 +1012,32 @@ public CloudServiceInstance getServiceInstance(String serviceName, boolean requi
                             .collect(Collectors.toList());
     }
 
-    private Map<String, Metadata> getServiceInstancesMetadata(List<String> names) {
+    private Map<String, Metadata> getServiceInstancesMetadata(List<String> serviceInstanceNames) {
```

Comment:
Rename to `getServiceInstancesMetadataInBatches`. The method that is currently named `getServiceInstancesMetadataWithoutBatching` should not know what batching is and should be called just `getServiceInstancesMetadata` since that's what it really does.